File: HomeWork05/Controller.java
public class Controller {
    Phonebook model;
    View view;

    public Controller(View view, Phonebook model) {
        this.view = view;
        this.model = model;
    }

    public void run(){

        new MakeMainMenuCommand(view).execute();
        while (true) {
            view.show();
            int numPunkt = view.getInt();
            PunktMenu pm = view.getMenu().getPunkt(numPunkt);
            if (pm == null) {
                view.setBar("Неправильный выбор");
            } else {
                pm.run();
            }
        }
        
        
    }

}

File: HomeWork07/Controllers/Controller.cs
namespace HomeWork07;

public class Controller {
    protected bool exitApp;
    protected Screen screen;
    protected Phonebook phonebook;
    protected View view;

    public Controller(Screen screen, Phonebook phonebook) {
        this.screen = screen;
        this.view = screen.getView();
        this.phonebook = phonebook;
        this.exitApp = false;
    }

    public void run() {

        new MakeMainMenuCommand(this).execute();
        while (!exitApp) {
            screen.show();
            int numPunkt = view.getPositiveInt();
            PunktMenu pm = screen.getMenu().getPunkt(numPunkt);
            if (pm == null) {
                screen.setBar("Неправильный выбор");
            } else {
                pm.run();
            }
        }

    }

    public Screen getScreen() {
        return this.screen;
    }

    public Phonebook getPhonebook() {
        return this.phonebook;
    }

    public void setExitApp() {
        this.exitApp = true;
    }

}
